src/microbe_stage/editor/tooltips/SelectionMenuToolTip.cs

```diff
@@ -213,12 +192,15 @@ public override void _ExitTree()
     /// <summary>
     ///   Instances the UI element for a modifier info
     /// </summary>
-    public void AddModifierInfo(string name, float value)
+    public void AddModifierInfo(string name, string value, float valueColor = 0, string? iconPath = null)
```

Comment:
I think `valueColor` is too unclear of a name, and doesn't follow our naming convention (should use "colour"). I suggest naming it something like `valueForColourApplying` to better describe what it does.